examples_test.go

```diff
@@ -92,13 +93,45 @@ c,3.0`
 	f := qframe.ReadCSV(strings.NewReader(input))
 	fmt.Println(f)
 	// Output:
-	// COL1(s) COL2(f)
+	// COL(s) COL0(f)
```

Comment:
Not sure what happened here but I can take a look at it. Perhaps the examples need to be added to the automated tests.